lib/declaration.ts

```diff
@@ -92,7 +95,7 @@ class DeclarationTranspiler extends base.TranspilerStep {
       case ts.SyntaxKind.PropertyDeclaration:
         var propertyDecl = <ts.PropertyDeclaration>node;
         this.visitDeclarationMetadata(propertyDecl);
-        var hasConstCtor = this.hasConstCtor(<ClassLike>propertyDecl.parent);
+        var hasConstCtor = this.hasAnnotation((<ClassLike>propertyDecl.parent).decorators, 'CONST');
```

Comment:
rename to hasConstClass? Also, at this point it might be smarter to extract a method.